src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.revrobotics.AbsoluteEncoder;
+import com.revrobotics.SparkMaxPIDController;
+import com.revrobotics.SparkMaxAbsoluteEncoder;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.config.Config;
+
+public class ArmSubsystem extends SubsystemBase {
+
+  private static final int SparkTopArmCANID = 0;
+  private static final int SparkBottomArmCANID = 0;
+  private static final MotorType motorType = MotorType.kBrushless;
+  private static final SparkMaxAbsoluteEncoder.Type encAbsType = SparkMaxAbsoluteEncoder.Type.kDutyCycle;
+  
+  private AbsoluteEncoder m_absoluteTopArmEncoder;
```

Comment:
We will use two CANCoders as absolute encoders.
You can remove these two absolute encoders.